main.go

```diff
@@ -91,9 +92,9 @@ var (
 	startTime                time.Time
 	stopAll                  uint32
 	measureLatency           bool
-	hdrLatencyFile  string
-	hdrLatencyUnits string
-	validateData    bool
+	hdrLatencyFile           string
```

Comment:
Formatting cleanups should be in separate commit. All of them.

Reply:
Done